cmd/src/cli/run.rs

```diff
@@ -0,0 +1,77 @@
+use cli::{self, package};
+use error::DefaultResult;
+use holochain_container_api::{config::*, container::Container};
+
+/// Starts a small container with the current application running
+pub fn run(package: bool, port: u16) -> DefaultResult<()> {
+    if package {
+        cli::package(true, Some(package::DEFAULT_BUNDLE_FILE_NAME.into()))?;
+    }
+
+    let agent_config = AgentConfiguration {
+        id: "hc-run-agent".into(),
+        key_file: "hc_run.key".into(),
+    };
+
+    let dna_config = DNAConfiguration {
+        id: "hc-run-dna".into(),
+        file: package::DEFAULT_BUNDLE_FILE_NAME.into(),
+        hash: "Qm328wyq38924ybogus".into(),
+    };
+
+    let instance_config = InstanceConfiguration {
+        id: "test-instance".into(),
+        dna: "hc-run-dna".into(),
+        agent: "hc-run-agent".into(),
+        logger: Default::default(),
+        storage: StorageConfiguration::Memory,
```

Comment:
I think we said it should be persistent. Then this should be set to file storage.

Reply:
Should it? Dunno, wasn't present to that conversation